### PR TITLE
Profiles

### PR DESCRIPTION
**Pull request by PhillippOhlandt**
It's not that beautiful but should work :)
I had to inline the styles for now because someone forgot to require foundation in packages.json ... :D 

You just have to install `ohlandt/user-profiles` after the deployment.

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ payoff: Sophisticated, Lightweight and Simple
 # The theme to use. Be sure to copy the {theme_path}/base-2014 folder, and
 # change the name here accordingly.
 #
-# Don't edit the templates in theme/base-2014, because they _will_ get updated
+# Don't edit the templates int theme/base-2014, because they _will_ get updated
```
Comment (bobdenotter) [inline app/config/config.yml]: Heheh. ;-)

File: theme/bolt-v20/news.twig [hunk 2]
```diff
@@ -11,7 +11,7 @@
         <article class="row">
             <h1><a href="{{item.link}}">{{item.title}}</a></h1>
             <h2 class="subheader">{{item.subtitle}}</h2>
-            <p class="meta" title="{{ item.datepublish|date("j F Y - H:i") }}">Published: {{ item.datepublish|date("l j F Y") }} by {{ item.user.displayname }}</p>
+            <p class="meta" title="{{ item.datepublish|date("j F Y - H:i") }}">Published: {{ item.datepublish|date("l j F Y") }} by <a  href="{{ profile_link(record.user) }}">{{ item.user.displayname }}</a></p>
```
Comment (bobdenotter) [inline theme/bolt-v20/news.twig]: Two spaces. </nitpick>

**Comment by bobdenotter** [comment]
@PhillippOhlandt If you fix the two minor typo's, i'll deploy this tomorrow! :-)

File: app/config/config.yml (hunk 1 above)
Comment (PhillippOhlandt) [inline app/config/config.yml]: wtf, sublime text went crazy!

File: theme/bolt-v20/news.twig (hunk 2 above)
Comment (PhillippOhlandt) [inline theme/bolt-v20/news.twig]: what? :D

Comment (bobdenotter) [inline theme/bolt-v20/news.twig]: `<a   href`

Comment (PhillippOhlandt) [inline theme/bolt-v20/news.twig]: ah

**Comment by bobdenotter** [comment]
whoop!

**Pull request merged**